docs/product/components/cards.html

```diff
@@ -1,73 +1,142 @@
 ---
 layout: page
 title: Cards
-description: A visual styling for grouping related information together within distinct containers.
+description: Cards are used to group similar concepts and tasks together to make information easier to scan, read, and act on. Cards should use a heading that sets clear expectations about the card’s purpose, paragraphs that put the most critical information first, and (optionally) calls to action on the bottom to direct user action.
 ---
 <section class="stacks-section">
     {% header h2 | Base %}
-    <p class="stacks-copy">The base card styling applies a border and box-shadow on hover.</p>
+    <p class="stacks-copy">The base card styling applies a border and padding to the card.</p>
+    <p class="stacks-copy">Cards can be any size and it's ok to increase the body text size for larger cards.</p>
     <div class="stacks-preview">
 {% highlight html linenos %}
 <article class="s-card">
-    <h1 class="fs-body3 fc-dark">…</h1>
-    <p class="fs-caption fc-light">…</p>
+    <h2 class="fs-body3 lh-sm fc-dark">…</h2>
+    <p class="fs-body1 fc-medium">…</p>
+    <button class="s-btn s-btn__primary s-btn__sm">…</button>
 </article>
 {% endhighlight %}
         <div class="stacks-preview--example">
-            <div class="grid">
-                <article class="wmx3 s-card">
-                    <h1 class="fs-body3 fc-dark">Sample card title</h1>
-                    <p class="fs-caption fc-light">This is some explanation copy to explain what to do.</p>
-                </article>
-            </div>
+            <article class="s-card wmx3 mb12">
+                <h2 class="fs-body3 lh-sm fc-dark">Base card title</h2>
+                <p class="fs-body1 fc-medium">This is a description of the card's content.</p>
+                <p class="grid gsx gs12">
+                    <button class="grid--cell s-btn s-btn__primary s-btn__sm">Call to action</button>
+                    <button class="grid--cell s-btn s-btn__sm">Cancel</button>
+                </p>
+            </article>
+            <article class="s-card">
+                <h2 class="fs-body3 lh-sm fc-dark">Base card title</h2>
+                <p class="fs-body2 fc-medium">This is a description of the card's content. Sometimes the description is two or three sentences.</p>
+                <p class="grid gsx gs12">
```

Comment:
Throughout Stacks, we generally use a grid spacing of `gs4` with buttons.

Reply:
done